README.md

```diff
@@ -32,6 +32,8 @@ General guidelines for Hellbender developers
 
 * Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
+* Each tool should have at least one good end-to-end integration test with a check for expected output, plus high-quality unit tests for all non-trivial utility methods/classes used by the tool. Although we have no specific coverage target, coverage should be extensive enough that if tests pass, the tool is guaranteed to be in a usable state.
```

Comment:
This list is super confusing -- instead of these cryptic abbreviations I'd recommend you edit it to assign one of the following values to each tool: "one integration test", "multiple integration tests", or "integration + unit tests", and for non-tool classes either "unit tests needed" or "no tests needed".

Also, do you agree with the requirement proposed above that each tool should have at a minimum one integration test?